packages/documentation/src/content/docs/integration/requirements/idp.mdx

```diff
@@ -12,179 +12,130 @@ import {
+| <Badge text="GET" variant="note" size="medium"/> | `/grant/{id}/{nonce}` | [Look up grant information](#look-up-grant-information) | Identity provider | No |
+| <Badge text="POST" variant="success" size="medium"/> | `/grant/{id}/{nonce}/{choice}` | [Accept or reject grant](#accept-or-reject-grant) | Identity provider | No |
+| <Badge text="GET" variant="note" size="medium"/> | `/interact/{id}/{nonce}/finish` | [Finish user interaction](#finish-interaction) | Identity provider | Yes |
+| <Badge text="POST" variant="success" size="medium"/> | `/interact/{id}/{nonce}` | [Continue grant](#continue-grant) | Open Payments client | Yes |
 
 </div>
 
-We also provide an <LinkOut href="https://github.com/interledger/rafiki/blob/main/packages/auth/src/openapi/specs/id-provider.yaml">OpenAPI specification</LinkOut> that describes the endpoints.
+We also provide an <LinkOut href="https://github.com/interledger/rafiki/blob/main/packages/auth/src/openapi/specs/id-provider.yaml">OpenAPI specification</LinkOut> that describes the endpoints. Note that the *Continue grant* endpoint is not included in the spec because it's part of the <LinkOut href="https://openpayments.dev/apis/auth-server/operations/post-continue/">Open Payments Auth Server API</LinkOut>.
 
 #### Start user interaction session
 
 Called by the client to establish an interactive session with your authorization server. Also redirects the client's browser session to your IdP consent screen.
 
 #### Look up grant information
 
-Called by your IdP server, and secured with an [`x-idp-secret`](#x-idp-secret-header) header, to get the list of access rights the client requested from your authorization server.
-
-The access rights are presented to the client's end-user on the consent screen. The authorization server's response is served on your defined [`INTERACTION_PORT`](#environment-variables).
+Called by your IdP server to retrieve a list of access rights, requested by the client, from your authorization server. The request is secured with an [`x-idp-secret`](#x-idp-secret-header) header. The access rights are presented to the client's end-user on the consent screen. The authorization server's response is served on your configured `INTERACTION_PORT`.
 
 #### Accept or reject grant
 
-Called by your IdP server, and secured with an [`x-idp-secret`](#x-idp-secret-header) header, to communicate the choice made by the end-user on the consent screen (accept/deny) to your authorization server.
-
-Your IdP server then redirects to the `finish` endpoint. The response is served on your defined [`INTERACTION_PORT`](#environment-variables)
+Your IdP server communicates the choice made by the end-user on the consent screen (accept/reject) to your authorization server. The request is secured with an [`x-idp-secret`](#x-idp-secret-header) header. Then, your IdP server calls the `finish` endpoint on Rafiki's `auth` service, which handles the interaction requests on your defined `INTERACTION PORT`.
```

Comment:
The part about `finish`, I'm not sure whether it belongs here on in the section below

Reply:
@mkurapov Do you have a preference for the wording here? One is specific to Rafiki and the other is more general to Open Payments:

Called by your IdP server to end the interaction and redirect the end-user's browser session to the...

* Option 1: `finish` endpoint on Rafiki's `auth` service, which handles the interaction requests on your defined `INTERACTION PORT`.
* Option 2: `finish` URI that was provided in the <LinkOut href="https://openpayments.dev/apis/auth-server/operations/post-request/">grant initialization request</LinkOut>.

Reply:
I think Option 2 is clear:

"Called by your IdP server to end the interaction. If there was a `finish` URI provided in the original grant initialization request, the authorization server redirects the user to that URI".

For the `INTERACTION_PORT`, I think it's sufficient to just mention it once in the "Interaction endpoints" section

Reply:
Also, for the "Your authorization server automatically redirects the end-user’s browser session to the finish endpoint on Rafiki’s auth service" sentence below, it seems as though "Your authorization server" and "Rafiki’s auth service" are different things perhaps